node/api/client/renter.go

```diff
@@ -55,6 +55,17 @@ func (c *Client) RenterPost(allowance modules.Allowance) (err error) {
 	return
 }
 
+// RenterPostRateLimit uses the /renter endpoint to change the renter's bandwidth rate
+// limit.
+func (c *Client) RenterPostRateLimit(readBPS, writeBPS int64, packetSize uint64) (err error) {
+	values := url.Values{}
+	values.Set("downloadspeed", strconv.FormatInt(readBPS, 10))
+	values.Set("uploadspeed", strconv.FormatInt(writeBPS, 10))
+	values.Set("packetsize", strconv.FormatUint(packetSize, 10))
```

Comment:
these need better names. `downloadspeed` doesn't imply that it's a maximum download speed. Also, I don't see these values being handled on the server-side?

Reply:
@lukechampine @DavidVorick let me know what you want them to be.
At the moment they are handled in `renterHandlerPost`.